dcs-liberation/dcs_liberation
# Issue 2059

This PR addresses Issue 2059 https://github.com/dcs-liberation/dcs_liberation/issues/2059 by adding the relevant yaml file. 

This PR also fixes an edge case found during testing where some unit types have a built in TGP in DCS and cannot mount one on a station.

Testing done on the above fix:

- Set up a loadout on an F15E with LGBs
- Observe that in the development built, with Restrict Weapons by Date enabled, the loadout falls back to dumb bombs
- Observe that in the PR,  with Restrict Weapons by Date enabled, no fall back occurs.
